bin/self-signed-certificate

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -eu
+
+if [ "$#" -eq 1 ]; then
+    DEST="$1"
+    cp /usr/local/share/ssl/key.pem $DEST
+    cp /usr/local/share/ssl/certificate.pem $DEST
+    exit 0
+elif [ "$#" -eq 6 ]; then
+    DEST="$1"
+    COUNTRY="$2"
+    LOCATION="$3"
+    ORG="$4"
+    ORG_UNIT="$5"
+    COMMON="$6"
+else
+    echo "Usage:"
+    echo "  self-signed-certificate DEST"
```

Comment:
What's the use case for this first kind of usage?

Reply:
Um, I have honestly no idea.

Reply:
OK, after removing it I found out why it was there and have added a little comment

Reply:
Ah yes, spotted it https://github.com/reside-ic/proxy-nginx/pull/1/files/10e614db2f149463b13440dcef0ab05db642f67f#diff-04c6e90faac2675aa89e2176d2eec7d8R30